src/Illuminate/Contracts/Mail/Mailer.php

```diff
@@ -38,4 +38,14 @@ public function raw($text, $callback);
      * @return \Illuminate\Mail\SentMessage|null
      */
     public function send($view, array $data = [], $callback = null);
+
+    /**
+     * Send a new message synchronously using a view.
+     *
+     * @param  \Illuminate\Contracts\Mail\Mailable|string|array  $mailable
+     * @param  array  $data
+     * @param  \Closure|string|null  $callback
+     * @return \Illuminate\Mail\SentMessage|null
+     */
+    public function sendNow($mailable, array $data = [], $callback = null);
```

Comment:
This is a breaking change. You can't modify or add methods to contracts in patch or minor releases.

Reply:
Thank you. I will close this and reopen on master.